### DISCUSSION TITLE
Tutorial for reproducing MPTrj dataset from materials project

### DISCUSSION THREAD

**Discussion by YutackPark** [CederGroupHub/chgnet#183] [Q&A] [answered]
Dear CHGNet developers,

Hi! I'm interested in the details of data parsing of MPTrj dataset. I have tried by following the instructions in the paper. However, I found that it is tricky to work with different docs of materials project API, and it was hard to determine appropriate hyperparameters (especially `StructureMatcher` and `EnergyMatcher`).

As the materials project keeps growing in its size, it would be nice for the community if there is a tutorial for reproducing MPTrj dataset, since MPTrj is popular and widely used in other GNN-IP models for various benchmarks.

**Comment by janosh** [comment, marked as answer]
>  it would be nice for the community if there is a tutorial for reproducing MPTrj dataset

we agree! have you seen [`QueryMPtrj.md`](https://github.com/CederGroupHub/chgnet/blob/2570b29f3/examples/QueryMPtrj.md)

**Reply by YutackPark** [reply]
How did I miss this? 😂 Sorry for the bother. The document is very clear and well-written. Thanks for your response!
